notifications.bs

```diff
@@ -739,35 +757,38 @@ method steps are:
 <ol>
  <li><p>Let <var>global</var> be the <a>current global object</a>.
 
- <li><p>Let <var>origin</var> be the <a>current settings object</a>'s
- <a for="environment settings object">origin</a>.
- <!-- Arguably this should snapshot origin somehow so that changes to it do not impact the
-      in parallel part. -->
+ <li><p>Let <var>permissionDescriptor</var> be the {{PermissionDescriptor}} with
+ {{PermissionDescriptor/name}} set to "<code>notifications</code>".
+
+ <li><p>Let <var>permissionStatus</var> be the result of
+ <a lt="create a PermissionStatus">creating a `PermissionStatus`</a> for
+ <var>permissionDescriptor</var>.
 
  <li><p>Let <var>promise</var> be <a for=/>a new promise</a> in <a>this</a>'s <a>relevant Realm</a>.
 
  <li>
   <p>Run these steps <a>in parallel</a>:
 
   <ol>
-   <li><p>Let <var>permission</var> be <a>permission</a> for <var>origin</var>.
+   <li><p>Run the <a for="powerful feature">permission query algorithm</a> with
+   <var>permissionDescriptor</var> and <var>permissionStatus</var>.
 
-   <li><p>If <var>permission</var> is "<code>default</code>", then ask the user whether showing
-   notifications for <var>origin</var> is acceptable. If it is, then set <var>permission</var> to
-   "<code>granted</code>"; otherwise "<code>denied</code>".
+   <li><p>Let <var>permission state</var> be <var>permissionStatus</var>'s
```

Comment:
Can we rename this to _permissionState_ throughout for consistency?

Reply:
Sure thing.